src/backend/src/prisma/migrations/20241121053030_allow_checklistitem_to_have_multiple_descriptions/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
can we delete the existing migration for onboarding and consolidate everything to one file